src/data/faq.json

```diff
@@ -95,15 +95,15 @@
                                 "anchor": "ramp_down_mailbox",
                                 "active": false,
                                 "textblock": [
-                                    "Until May 31, 2023 there will be community support, in which incoming messages will be read and answered if applicable."
+                                    "The community support, which includes reading and answered if applicable to messages received at <a href='corona-warn-app.opensource...sap...com' class='email'>corona-warn-app.opensource...sap...com</a>, will continue until May 31, 2023. After that, the mailbox <a href='CoronaWarnApp...RKI...de' class='email'>CoronaWarnApp...RKI...de</a> will still be available, which is planned to be maintained until the end of 2023."
                                 ]
                             },
                             {
                                 "title": "What happens with my personal data in the emails?",
                                 "anchor": "ramp_down_mailbox_data",
                                 "active": false,
                                 "textblock": [
-                                    "An automatic deletion is set up for the CWA mailbox: All emails that are now in our CWA mailbox or still reach that mailbox will be deleted irreversibly without replacement, after 6 months at the latest, to ensure the protection of your data."
+                                    "An automatic deletion is set up for the CWA mailbox from SAP (<a href='corona-warn-app.opensource...sap...com' class='email'>corona-warn-app.opensource...sap...com</a>): All emails that are now in our CWA mailbox or still reach that mailbox will be deleted irreversibly without replacement, after 6 months at the latest, to ensure the protection of your data."

```

Comment:
"from SAP" => 
"of SAP"

"will be deleted irreversibly without replacement" =>
"will be completely deleted" (as translation of "vollständig gelöscht")

Reply:
"An automatic deletion" =>
"Automatic deletion"

Reply:
```suggestion
                                    "Automatic deletion is set up for the CWA mailbox of SAP (<a href='corona-warn-app.opensource...sap...com' class='email'>corona-warn-app.opensource...sap...com</a>): All emails that are now in our CWA mailbox or still reach that mailbox will be completely deleted, after 6 months at the latest, to ensure the protection of your data."
```